package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@godaddy/split-javascript-data-loader",
```

Comment:
for https://github.secureserver.net/hivemind/hivemind-sdk-node we decided to leave `node` out of the package name. I think it makes sense to do the same here and just fall the package `split-data-loader`

Reply:
We didn't do that for the [node-serializer](https://github.com/godaddy/split-node-serializer/blob/master/package.json#L2). Is that something we should be changing there also?